src/ruby-gem/spec/twsearch_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+RSpec.describe Twsearch do
+  it "has a version number" do
+    expect(described_class::VERSION).not_to be_nil
+  end
+
+  describe ".random_scramble_for_event" do
+    subject(:tz_name) { described_class.random_scramble_for_event(event_id) }
```

Comment:
I can't find anything about a `tz_name` convention in Ruby. Is this a holdover from pasted code?

Reply:
Yes, whoops :see_no_evil: Sorry! I used https://github.com/HarlemSquirrel/tzf-rb as "role model" for the Rust bindings, and I evidently also copied (a bit too much) of the spec code

Reply:
Fixed now